Cargo.toml

```diff
@@ -47,16 +53,15 @@ tokio = { version = "1.40.0", features = ["rt-multi-thread"] }
 op-alloy-consensus = { version = "0.2.7", optional = true }
 op-alloy-network = { version = "0.2.7", optional = true }
 op-alloy-rpc-types = { version = "0.2.7", optional = true }
+clap = { version = "4.5.17", features = ["derive"] }
```

Comment:
Can you group CLI dependencies like `RPC Storage dependencies`? In the future, we should make them `optional`, the CLI a package or use Cargo workspace to make the core library as lean as possible.